django/utils/url_config.py

```diff
@@ -0,0 +1,65 @@
+import inspect
+from collections import defaultdict
+
+from django.core.exceptions import ImproperlyConfigured
+from django.utils import module_loading
+
+BACKENDS = defaultdict(dict)
+
+
+def add_backend(backend_type, scheme, path):
```

Comment:
This function's name starts with "add" but the convenience functions' names start with "register".  Would it help to keep these uniform?

Reply:
👍